src/main/java/com/google/sps/data/Card.java

```diff
@@ -3,15 +3,16 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
 
 public final class Card {
 
-    private String id;
     private String blobKey;
     private String labels;
     private String fromLang;
     private String toLang;
-    private String textNotTranslated;
+    private String rawText;
     private String textTranslated;
     private String cardKey;
```

Comment:
Just call this `key`. Prefixing it with `card` makes it more confusing because we're already in the card class — is `cardKey` the key for the current card, or does it reference some other card?